demo/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The "demo" name is misleading, the name should suggest its purpose (test-bed, system-under-test, etc...)